Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/ExpectedOutputs/SEC102_001.EmailAddress.sarif

```diff
@@ -256,8 +297,13 @@
           }
         },
         {
-          "ruleId": "SEC102/001",
-          "ruleIndex": 0,
+          "rule": {
```

Comment:
>rule [](http://example.com/codeflow?start=11&length=4)

everyone now uses a ReportingDescriptorReference to point to the proper extension that has rules metadata.